src/generated/java/com/prowidesoftware/swift/model/mt/MtSequenceInfoEnum.java

```diff
@@ -0,0 +1,131 @@
+    MTMT516("A,B,C"),
+    MTMT517("A,A/A1"),
+    MTMT518("A,A/A1,B,B/B1,B/B2,C,C/C1,C/C2,C/C3,D,E"),
+    MTMT519("A,A/A1,B,B/B1,B/B2,C,C/C1,C/C2,D"),
+    MTMT524("A,A/A1,B,B/B1,C"),
+    MTMT526("A,B"),
+    MTMT527("A,A/A1,A/A2,B,B/B1,C,D,E"),
+    MTMT530("A,B,B/B1,C,C/C1,C/C1/C1a"),
+    MTMT535("A,A/A1,B,B/B1,B/B1/B1a,B/B1/B1b,B/B1/B1b/B1b1,B/B1/B1c,C"),
+    MTMT536("A,A/A1,B,B/B1,B/B1/B1a,B/B1/B1a/B1a1,B/B1/B1a/B1a2,B/B1/B1a/B1a2/B1a2A,C"),
+    MTMT537("A,A/A1,B,B/B1,B/B2,B/B2/B2a,B/B2/B2b,B/B2/B2b/B2b1,C,C/C1,C/C2,C/C2/C2a,C/C3,C/C3/C3a,D,D/D1,D/D1/D1a,D/D1/D1a/D1a1,D/D1/D1a/D1a1/D1a1A,D/D1/D1a/D1a1/D1a1A/D1a1A1,D/D1/D1a/D1a1/D1a1B,D/D1/D1a/D1a1/D1a1B/D1a1B1,D/D1/D1a/D1a1/D1a1B/D1a1B1/D1a1B1a,D/D1/D1a/D1a1/D1a1B/D1a1B1/D1a1B1a/D1a1B1a1,E"),
+    MTMT538("A,A/A1,B,B/B1,B/B2,B/B2/B2a,B/B2/B2a/B2a1,C"),
+    MTMT540("A,A/A1,B,B/B1,C,C/C1,D,E,E/E1,E/E2,E/E3,F"),
+    MTMT541("A,A/A1,B,B/B1,C,C/C1,D,E,E/E1,E/E2,E/E3,F"),
+    MTMT542("A,A/A1,B,B/B1,C,C/C1,D,E,E/E1,E/E2,E/E3,F"),
+    MTMT543("A,A/A1,B,B/B1,C,C/C1,D,E,E/E1,E/E2,E/E3,F"),
+    MTMT544("A,A/A1,B,B/B1,C,C/C1,D,E,E/E1,E/E2,E/E3,F"),
+    MTMT545("A,A/A1,B,B/B1,C,C/C1,D,E,E/E1,E/E2,E/E3,F"),
+    MTMT546("A,A/A1,B,B/B1,C,C/C1,D,E,E/E1,E/E2,E/E3,F"),
+    MTMT547("A,A/A1,B,B/B1,C,C/C1,D,E,E/E1,E/E2,E/E3,F"),
+    MTMT548("A,A/A1,A/A2,A/A2/A2a,B,B/B1,C,C/C1,C/C1/C1a,C/C1/C1a/C1a1,C/C1/C1a/C1a1/C1a1A,C/C1/C1a/C1a1/C1a1A/C1a1A1,C/C1/C1a/C1a1/C1a1B,C/C1/C1a/C1a1/C1a1B/C1a1B1,C/C1/C1a/C1a1/C1a1B/C1a1B1/C1a1B1a,C/C1/C1a/C1a1/C1a1B/C1a1B1/C1a1B1a/C1a1B1a1,D"),
+    MTMT549("A,A/A1,B,C,D"),
+    MTMT558("A,A/A1,A/A2,A/A2/A2a,A/A3,B,B/B1,C,D,E"),
+    MTMT564("A,A/A1,B,B/B1,B/B2,C,D,E,E/E1,E/E1/E1a,E/E2,F"),
+    MTMT565("A,A/A1,B,B/B1,B/B2,C,D,E"),
+    MTMT566("A,A/A1,B,B/B1,C,D,D/D1,D/D1/D1a,D/D1/D1b,D/D2,D/D2/D2a,D/D2/D2b,E"),
+    MTMT567("A,A/A1,A/A2,A/A2/A2a,B,C"),
+    MTMT568("A,A/A1,B,B/B1,C"),
+    MTMT569("A,A/A1,A/A2,B,C,C/C1,C/C1/C1a,C/C1/C1a/C1a1,C/C1/C1a/C1a1/C1a1A,D"),
+    MTMT575("A,A/A1,B,B/B1,B/B1/B1a,B/B1/B1a/B1a1,B/B1/B1a/B1a2,B/B1/B1a/B1a3,B/B1/B1a/B1a4,C,C/C1,C/C2,C/C2/C2a,D"),
+    MTMT576("A,A/A1,B,B/B1,B/B2,B/B2/B2a,B/B2/B2b,B/B2/B2c,C"),
+    MTMT578("A,A/A1,B,B/B1,C,C/C1,D,E,E/E1,E/E2,E/E3,F"),
+    MTMT586("A,A/A1,B,B/B1,B/B2,B/B3,B/B4,B/B4/B4a,B/B5,B/B6,B/B6/B6a,B/B6/B6b,C"),
+    MTMT600("A,B,C,D,D/D1,D/D1/D1a,D/D1/D1a/D1a1"),
+    MTMT601("A,B,B/B1,B/B1/B1a,B/B1/B1a/B1a1"),
+    MTMT620("A,B,C,D,E,F,G,G/G1,G/G2,H"),
+    MTMT670("A,A/A1,A/A2,B,B/B1,B/B2,C"),
+    MTMT671("A,A/A1,B,B/B1,B/B2,C"),
+    MTMT760("A,B,C"),
+    MTMT767("A,B,C");
+
+    private final String sequencesInfo;
+
+    MtSequenceInfoEnum(String sequencesInfo) {
+        this.sequencesInfo = sequencesInfo;
+    }
+
+    /**
+    * Returns the sequence paths for this MT schema.
+    * Each sequence path represents a hierarchical structure within the MT schema.
+    *
+    * @return A comma-separated string of sequence paths.
+    *
+    */
+    public String getSequencesInfo() {
```

Comment:
as convention for enum properties it would be "sequencesInfo()" not a getter. And make this method return already a Set<String>. You can define the property as Set and create the set in the constructor. Or you can keep it as String in the property field and create the Set dynamically in this method.

Reply:
Ok